pandas/conftest.py

```diff
@@ -1140,9 +1140,9 @@ def all_numeric_accumulations(request):
 @pytest.fixture
 def strict_data_files(pytestconfig):
```

Comment:
Should the naming of this fixture change since it checks `no-strict-data-files`?

Reply:
With store_false, argparse passes in False if ``no-strict-data-files`` is passed in, and True otherwise, so this fixture should still be correct.